kernel/src/checkpoint/mod.rs

```diff
@@ -34,7 +34,7 @@
 //! # use delta_kernel::checkpoint::CheckpointDataIterator;
```

Comment:
ah looks like some of the docs are broken due to no longer pulling in `Snapshot`. and btw you can test with `cargo doc` and/or `cargo doc --open` (latter is just a nice shortcut to pull up the docs we build after they are compiled)

we could just do something like add this at the bottom:

```
[`Snapshot::checkpoint`]: crate::Snapshot::checkpoint
```